crates/astria-core/src/protocol/transaction/v1alpha1/mod.rs

```diff
@@ -198,7 +203,12 @@ impl SignedTransaction {
 
     #[must_use]
     pub fn actions(&self) -> &[Action] {
-        &self.transaction.actions
+        self.transaction.actions.actions()
+    }
+
+    #[must_use]
+    pub fn group(&self) -> Option<ActionGroup> {
```

Comment:
Noted elsewhere: I think we can avoid exposing this publicly for the time being by providing `contains_xxx_group_actions` or `is_xxx_group_actions`